tests/lib/tools/snaps-state

```diff
@@ -59,6 +62,92 @@ install_local_as() {
     install_local "$snap" --name "$name" "$@"
 }
 
+is_classic_confinement_supported() {
```

Comment:
I see you want this badly :-)

Reply:
This will be diferent than the previous one, it will be used to query the type of confinement, so we can reuse the new command on different scenarios. This will be included in the next commit.